### PR TITLE
Bug 2091658: Improve the sriov gather script

### PR DESCRIPTION
**Pull request by liornoy**
Added improvments to the sriov gather script:
1. Use the SRIOV_NS var instead of static: "openshift-sriov-network-operator".
2. In the gather_ethtool function, do not log interfaces with @ in the name.

### CODE REVIEW

**Comment by liornoy** [comment]
@SchSeba please review =)

**Review by SchSeba — COMMENTED** [review]
LGTM!

Thanks!

File: collection-scripts/gather_sriov [hunk 1]
```diff
@@ -47,19 +47,19 @@ function gather_ethtool(){
...
   /bin/bash -c "ip -o link show 2>/dev/null")"
 
   # Cut long interfaces names.
-  INTERFACES="$(echo "$OUT" | awk -F': ' '{print $2}' | cut -d '@' -f 1)"
+  INTERFACES="$(echo "$OUT" | awk -F': ' '{print $2}')"
 
   # Run ethtool for each interface except for lo.
   while IFS= read -r interface; do
-    if [ -z "$interface" ] || [ "$interface" = "lo" ]; then
+    if [ -z "$interface" ] || [ "$interface" = "lo" ] || [[ "$interface" == *"@"* ]]; then
```
Comment (bn222) [inline collection-scripts/gather_sriov:58]: why?

Comment (SchSeba) [reply collection-scripts/gather_sriov:58]: we want to remove all the veth (ovn-k) related interfaces.

there is no need for sriov must-gather to get that information.

**Comment by zeeke** [comment]
@RickJWagner can you please (if you have the permissions) change the title of this PR in

`Bug 2091658: Improve the sriov gather script`

as a BZ is needed for it to get merged

**Comment by RickJWagner** [comment]
Hi @zeeke 
I've made a change to the title of https://bugzilla.redhat.com/show_bug.cgi?id=2091658 to match this issue tracker.
Please have a look, let me know if it needs adjusted, etc.

**Comment by zeeke** [comment]
/retitle Bug 2091658: Improve the sriov gather script

**Pull request merged**